ext/alias_editor/theme.php

```diff
@@ -30,7 +30,11 @@ public function display_aliases(HTMLElement $table, HTMLElement $paginator): voi
         $page->set_title("Alias List");
         $page->set_heading("Alias List");
         $page->add_block(new NavBlock());
-        $page->add_block(new Block("Aliases", $html));
+		$block = new Block("Aliases", $html);
+		$block->body = str_replace(array("name='c_oldtag'", "name='c_newtag'"), 
+						array("name='c_oldtag' class='autocomplete_tags'", "name='c_newtag' class='autocomplete_tags'"), 
+						$block->body);
+        $page->add_block($block);
```

Comment:
Doing search & replace on generated HTML is very fragile - I'm guessing that the intention here is to add `class='autocomplete_tags'` to the "add alias" inputs? I'd do that by going to the `AliasTable` definition and replacing `TextColumn` with `TagColumn` (a new class which subclasses `TextColumn` and overrides the `create_input()` method)

Reply:
Which directory would you like me to define a TagColumn class?

Reply:
That is a good question - I would put it in `core/imageboard/tagcolumn.php` (`core` because it is something that seems generally useful for multiple extensions, and `imageboard` because that's where tag-management code currently lives)